lib/govuk_message_queue_consumer/consumer.rb

```diff
@@ -14,16 +14,15 @@ class Consumer
 
     # Create a new consumer
     #
-    # @param queue_name [String] Your queue name. This is specific to your application.
     # @param exchange_name [String] Name of the exchange to bind to, for example `published_documents`
+    # @param queue_name [String] Your queue name. This is specific to your application,
+    #                            and should already exist and have a binding via puppet
     # @param processor [Object] An object that responds to `process`
-    # @param routing_key [String] The RabbitMQ routing key to bind the queue to
     # @param statsd_client [Statsd] An instance of the Statsd class
-    def initialize(queue_name:, exchange_name:, processor:, routing_key: '#', statsd_client: NullStatsd.new)
-      @queue_name = queue_name
+    def initialize(exchange_name:, queue_name:, processor:, statsd_client: NullStatsd.new)
```

Comment:
Not especially - just a passing thought to order the same as the hierarchy of message flow in rabbitmq